package/.pylintrc

```diff
@@ -199,7 +199,6 @@ enable=abstract-class-instantiated,
        print-statement,
        xrange-builtin,
        zip-builtin-not-iterating,
-       map-builtin-not-iterating,
```

Comment:
That check is erroring out the build. Do you know of any alternate way to write the labdas Ive written?